### PR TITLE
Loading the user session on the privacy page.

### PR DESCRIPTION
**Pull request by jonursenbach**
As the title says. When logged in, you'll now see the nav and no longer asked to log in again at the bottom of <https://bigboringsystem.com/privacy>.

### CODE REVIEW

File: views/privacy.jade [hunk 1]
```diff
@@ -106,4 +106,5 @@ block content
   article
     p If you have found a security vulnerability in BBS or have a security incident to report, please email us at bigboringsystem@gmail.com. Please include a detailed summary of the issue you've discovered. It'd be helpful if you can also include an email address where we can reach you if we need more information.
 
-  h2 enter your phone number to sign in
+  if !session
```
Comment (ednapiranha) [inline views/privacy.jade]: you can actually just set `block login` as empty here instead and it'll take care of it. like in this example https://github.com/bigboringsystem/bigboringsystem/blob/master/views/post.jade#L36 (overrides the default in layout.jade)

Comment (jonursenbach) [inline views/privacy.jade]: Oh, rad. Updated.

**Comment by ednapiranha** [comment]
wait where did the block go?

**Comment by jonursenbach** [comment]
Sorry, it's late and I'm really klutzing this up. I reverted to my original commit.

Adding an empty login block on the private page results in no login box being shown on the privacy page. 

2015-01-09 at 11 02 pm

Diff:
```
diff --git a/views/privacy.jade b/views/privacy.jade
index 445d4d1..1429f39 100644
--- a/views/privacy.jade
+++ b/views/privacy.jade
@@ -106,5 +106,4 @@ block content
   article
     p If you have found a security vulnerability in BBS or have a security incident to report, please email us at bigboringsystem@gmail.com. Please include a 
 
-  if !session
-    h2 enter your phone number to sign in
+block login
```

Even with just using the default login block, there's still the `enter your phone number to sign in` header. Could move that into the layout and show that if we're on the privacy page, but that seems like and unnecessary logic to have in there.

**Comment by isaacs** [comment]
@jonursenbach The view diff is fine.  But you do still need to give it a session context value.

Warning, untested, but this might fix it:

```patch
diff --git a/lib/services.js b/lib/services.js
index b1217a3..57fc831 100644
--- a/lib/services.js
+++ b/lib/services.js
@@ -131,5 +131,8 @@ exports.profile = function (request, reply) {
 };
 
 exports.privacy = function(request, reply) {
-    reply.view('privacy');
+  var context = {
+    session: request.session.get('uid')
+  };
+  reply.view('privacy', context);
 };
diff --git a/views/privacy.jade b/views/privacy.jade
index 7b42a20..ec59b4d 100644
--- a/views/privacy.jade
+++ b/views/privacy.jade
@@ -106,4 +106,4 @@ block content
   article
     p If you have found a security vulnerability in BBS or have a security incident to report, please email us at bigboringsystem@gmail.com. Please include a detailed summary of the issue you've discovered. It'd be helpful if you can also include an email address where we can reach you if we need more information.
 
-  h2 enter your phone number to sign in
+block login
```

**Comment by jasonrhodes** [comment]
Does:

```
if session
  block login
```

work? Not sure if you can include blocks in if branches but that's what you want, I think—override default login block if user is already logged in. 

If that doesn't work then just `block login` for everyone seems fine, you don't have to be able to login from a privacy page (maybe it's even weird to have the login there ever, now that I think about it?)

**Comment by isaacs** [comment]
@jasonrhodes You still have to pass the `session` key on the context object, and since `login.jade` already forks based on the session, that fixes the issue.

**Comment by jasonrhodes** [comment]
Ah! It's in `layout.jade` but yeah, it checks for the session. So if you want the login form on the privacy page, you actually _don't_ want to include `block login` in `privacy.jade`, just let the default login block come through from `layout.jade`. 

Make sense, @jonursenbach ?

**Comment by isaacs** [comment]
Right, becasue `extends layout`.  Got it.

**Comment by jasonrhodes** [comment]
@jonursenbach did you want to add the `block login` back in and rebase? Then I can take another look late tonight, or whenever.

**Comment by jonursenbach** [comment]
@jasonrhodes Sorry, been really busy with the 9-5. Will probably get back to this tonight or tomorrow.

**Comment by jasonrhodes** [comment]
@jonursenbach turns out I'm an idiot and said the opposite of what I meant there in that last comment. I'll just rebase this and get it merged for you so we can end the confusion, haha. Thanks for the PR!

**Comment by jonursenbach** [comment]
@jasonrhodes Sorry I wasn't able to finish it up, but thanks for merging my initial work in. :pizza:

**Pull request closed without merging**